nsv13/code/modules/power/gravitygenerator_modular.dm

```diff
@@ -0,0 +1,73 @@
+///This is the modular NSV file for gravity generators, to keep the base file more clear of conflicts.
+
+//Modular proc attachment
+/obj/machinery/gravity_generator/main/attackby(obj/item/I, mob/user, params)
+	if(!(obj_flags & EMAGGED))
+		return ..()
+	if(I?.tool_behaviour != TOOL_MULTITOOL)
+		return ..()
+	if(on || charge_count > 0 || charging_state == POWER_UP)
+		to_chat(user, "<span class='warning'>You should not mess with this device's circuitry while it is active!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>You should not mess with the [src]'s circuitry while it is active!</span>")
```
To stay consistent with the rest

Reply:
Reasonable, although 'the' is not needed thanks to the interpreter.